### ISSUE TITLE
Build failing (without code change)

### ISSUE THREAD

**Issue by breml** [logstash-plugins/logstash-codec-cef#16]
### TLDR;
I think this line is not working with the new version of logstash-core-event:
https://github.com/logstash-plugins/logstash-codec-cef/blob/master/lib/logstash/codecs/cef.rb#L74

@logstash-core what do you think?

### Long version
After committing a documentation change, the build is failing (http://build-eu-00.elastic.co/view/LS%20Plugins/view/LS%20Codecs/job/logstash-plugin-codec-cef-unit/).
This is due to changed dependencies.

Gemfile.lock.old (a few days ago):
```
PATH
  remote: .
  specs:
    logstash-codec-cef (2.1.0)
      logstash-core (>= 2.0.0.beta2, < 3.0.0)

GEM
  remote: https://rubygems.org/
  specs:
    cabin (0.7.2)
    clamp (0.6.5)
    coderay (1.1.1)
    concurrent-ruby (0.9.2-java)
    diff-lcs (1.2.5)
    ffi (1.9.10-java)
    filesize (0.0.4)
    gem_publisher (1.5.0)
    gems (0.8.3)
    i18n (0.6.9)
    insist (1.0.0)
    jrjackson (0.3.8)
    jruby-openssl (0.9.13-java)
    kramdown (1.10.0)
    logstash-core (2.2.2-java)
      cabin (~> 0.7.0)
      clamp (~> 0.6.5)
      concurrent-ruby (= 0.9.2)
      filesize (= 0.0.4)
      gems (~> 0.8.3)
      i18n (= 0.6.9)
      jrjackson (~> 0.3.7)
      jruby-openssl (= 0.9.13)
      logstash-core-event (~> 2.2.2)
      minitar (~> 0.5.4)
      pry (~> 0.10.1)
      rubyzip (~> 1.1.7)
      stud (~> 0.0.19)
      thread_safe (~> 0.3.5)
      treetop (< 1.5.0)
    logstash-core-event (2.2.2-java)
    logstash-devutils (0.0.18-java)
      gem_publisher
      insist (= 1.0.0)
      kramdown
      minitar
      rake
      rspec (~> 3.1.0)
      rspec-wait
      stud (>= 0.0.20)
    method_source (0.8.2)
    minitar (0.5.4)
    polyglot (0.3.5)
    pry (0.10.3-java)
      coderay (~> 1.1.0)
      method_source (~> 0.8.1)
      slop (~> 3.4)
      spoon (~> 0.0)
    rake (11.0.1)
    rspec (3.1.0)
      rspec-core (~> 3.1.0)
      rspec-expectations (~> 3.1.0)
      rspec-mocks (~> 3.1.0)
    rspec-core (3.1.7)
      rspec-support (~> 3.1.0)
    rspec-expectations (3.1.2)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.1.0)
    rspec-mocks (3.1.3)
      rspec-support (~> 3.1.0)
    rspec-support (3.1.2)
    rspec-wait (0.0.8)
      rspec (>= 2.11, < 3.5)
    rubyzip (1.1.7)
    slop (3.6.0)
    spoon (0.0.4)
      ffi
    stud (0.0.22)
    thread_safe (0.3.5-java)
    treetop (1.4.15)
      polyglot
      polyglot (>= 0.3.1)

PLATFORMS
  java

DEPENDENCIES
  logstash-codec-cef!
  logstash-devutils

BUNDLED WITH
   1.11.2
```

Gemfile.lock  (today):
```
PATH
  remote: .
  specs:
    logstash-codec-cef (2.1.0)
      logstash-core (>= 2.0.0.beta2, < 3.0.0)

GEM
  remote: https://rubygems.org/
  specs:
    cabin (0.8.1)
    clamp (0.6.5)
    coderay (1.1.1)
    concurrent-ruby (0.9.2-java)
    diff-lcs (1.2.5)
    ffi (1.9.10-java)
    filesize (0.0.4)
    gem_publisher (1.5.0)
    gems (0.8.3)
    i18n (0.6.9)
    insist (1.0.0)
    jar-dependencies (0.3.2)
    jrjackson (0.3.8)
    jruby-openssl (0.9.13-java)
    kramdown (1.10.0)
    logstash-core (2.3.0.snapshot1-java)
      cabin (~> 0.8.0)
      clamp (~> 0.6.5)
      concurrent-ruby (= 0.9.2)
      filesize (= 0.0.4)
      gems (~> 0.8.3)
      i18n (= 0.6.9)
      jrjackson (~> 0.3.7)
      jruby-openssl (= 0.9.13)
      logstash-core-event-java (= 2.3.0.snapshot1)
      minitar (~> 0.5.4)
      pry (~> 0.10.1)
      rubyzip (~> 1.1.7)
      stud (~> 0.0.19)
      thread_safe (~> 0.3.5)
      treetop (< 1.5.0)
    logstash-core-event-java (2.3.0.snapshot1-java)
      jar-dependencies
      ruby-maven (~> 3.3.9)
    logstash-devutils (0.0.18-java)
      gem_publisher
      insist (= 1.0.0)
      kramdown
      minitar
      rake
      rspec (~> 3.1.0)
      rspec-wait
      stud (>= 0.0.20)
    method_source (0.8.2)
    minitar (0.5.4)
    polyglot (0.3.5)
    pry (0.10.3-java)
      coderay (~> 1.1.0)
      method_source (~> 0.8.1)
      slop (~> 3.4)
      spoon (~> 0.0)
    rake (11.1.1)
    rspec (3.1.0)
      rspec-core (~> 3.1.0)
      rspec-expectations (~> 3.1.0)
      rspec-mocks (~> 3.1.0)
    rspec-core (3.1.7)
      rspec-support (~> 3.1.0)
    rspec-expectations (3.1.2)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.1.0)
    rspec-mocks (3.1.3)
      rspec-support (~> 3.1.0)
    rspec-support (3.1.2)
    rspec-wait (0.0.8)
      rspec (>= 2.11, < 3.5)
    ruby-maven (3.3.10)
      ruby-maven-libs (~> 3.3.1)
    ruby-maven-libs (3.3.3)
    rubyzip (1.1.7)
    slop (3.6.0)
    spoon (0.0.4)
      ffi
    stud (0.0.22)
    thread_safe (0.3.5-java)
    treetop (1.4.15)
      polyglot
      polyglot (>= 0.3.1)

PLATFORMS
  java

DEPENDENCIES
  logstash-codec-cef!
  logstash-devutils

BUNDLED WITH
   1.11.2
```

Diff:
```
$ diff Gemfile.lock Gemfile.lock.old
10c10
<     cabin (0.8.1)
---
>     cabin (0.7.2)
21d20
<     jar-dependencies (0.3.2)
25,26c24,25
<     logstash-core (2.3.0.snapshot1-java)
<       cabin (~> 0.8.0)
---
>     logstash-core (2.2.2-java)
>       cabin (~> 0.7.0)
34c33
<       logstash-core-event-java (= 2.3.0.snapshot1)
---
>       logstash-core-event (~> 2.2.2)
41,43c40
<     logstash-core-event-java (2.3.0.snapshot1-java)
<       jar-dependencies
<       ruby-maven (~> 3.3.9)
---
>     logstash-core-event (2.2.2-java)
61c58
<     rake (11.1.1)
---
>     rake (11.0.1)
76,78d72
<     ruby-maven (3.3.10)
<       ruby-maven-libs (~> 3.3.1)
<     ruby-maven-libs (3.3.3)
```

Could it be, that the new logstash-core-event does not work with the following line: 
https://github.com/logstash-plugins/logstash-codec-cef/blob/master/lib/logstash/codecs/cef.rb#L74

**Comment by breml** [comment]
Closed by #18 and #19
